io.github.buonhobo.KatharaGUI.metainfo.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Alex Bonini -->
+<component type="desktop">
+  <id>io.github.buonhobo.KatharaGUI</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <name>KatharaGUI</name>
+  <summary>A simple Kathara GUI to make experimentation easier</summary>
+
+  <developer_name>Alex Bonini</developer_name>
+  <developer id="io.github.buonhobo">
+    <name>Alex Bonini</name>
+  </developer>
+
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <display_length compare="ge">360</display_length>
+  </recommends>
+
+  <description>
+
+    <p> KatharaGUI is a simple GUI for the Kathara network emulation tool.                           </p>
+    <p> It allows you to start and stop labs and connect to the containers in a convenient way.      </p>
+    <p> KatharaGUI requires Docker to be installed and correctly configured on your system.          </p>
+
+    <p> This app is written in Python and uses GTK 4, libadwaita, and VTE.                           </p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>KatharaGUI when connected to a machine</caption>
+      <image>https://github.com/BuonHobo/KatharaGUI/blob/main/data/screenshots/LFGFFaB.png?raw=true</image>
```

Comment:
these needs to be direct raw.githubusercontet.com links no html pages. Use a commit or a tag in place of `main` in the link.

Reply:
same for below.